frontend/src/components/TextEditorPopup.tsx

```diff
@@ -102,7 +102,7 @@ const TextEditorPopup = (props: DialogProps) => {
         >
           <ContentEditor
             content={newTextContent}
-            placeholder={t(StringBank.INSERT_NEW_VERSION)}
+            placeholder={newTextContent ? newTextContent.text : t(StringBank.INSERT_NEW_VERSION)}
```

Comment:
```suggestion
            placeholder={t(StringBank.INSERT_NEW_VERSION)}
```

Reply:
It is the content, just not the placeholder

Reply:
but the placeholder is what is shown.
anyway, I changed as requested